rubocop-airbnb/config/rubocop-layout.yml

```diff
@@ -227,7 +227,7 @@ Layout/FirstMethodParameterLineBreak:
   Enabled: false
 
 # Supports --auto-correct
-Layout/FirstParameterIndentation:
+Layout/IndentFirstArgument:
```

Comment:
changing these rule names seems like a breaking change. why?

Reply:
I copied the new yml configs from https://github.com/airbnb/ruby/pull/150. If the older ones are forwards compatible, then there's no reason to do that (should have checked!)

Reply:
They are not forwards compatible:
```
Error: The `Layout/FirstParameterIndentation` cop has been renamed to `Layout/IndentFirstArgument`.
(obsolete configuration found in config/0.76/rubocop-layout.yml, please update it)
The `Layout/IndentArray` cop has been renamed to `Layout/IndentFirstArrayElement`.
(obsolete configuration found in config/0.76/rubocop-layout.yml, please update it)
The `Layout/IndentHash` cop has been renamed to `Layout/IndentFirstHashElement`.
(obsolete configuration found in config/0.76/rubocop-layout.yml, please update it)
```

Reply:
These two commits make it backwards compatible by splitting what config is used based on ruby version:
https://github.com/airbnb/ruby/pull/162/commits/aaf5dd82f13ca962c7cc306a79ac92e3bd99e844
https://github.com/airbnb/ruby/pull/162/commits/1c0591dad875547c1522902bb4c6a5a68581149e

That said it's a bit of a hack